firstvoices/backend/views/import_job_views.py

```
@@ -159,3 +185,24 @@ def confirm(self, request, site_slug=None, pk=None):
         return Response(
             serializer.data, status=status.HTTP_202_ACCEPTED, headers=headers
         )
+
+    @action(detail=True, methods=["post"])
+    def validate(self, request, site_slug=None, pk=None):
+        """
+        Method to start the validation process on a given import-job.
+        """
+        import_job_id = self.kwargs["pk"]
```

If there is already a validation in progress from the initial creation, what happens? Should we return a 400 error at this point? We don't have a way to cancel the first validation, and there is no guarantee the two reports will complete in the right order. I think we need to think through the collision cases a bit more

---

With the changes I just added, if the `/validate` endpoint is hit while a job is queued or running (same job or a different job), it will give a 400 and state that the user should wait for the current job to finish.